app/bundles/PageBundle/EventListener/PointSubscriber.php

```diff
@@ -43,7 +44,7 @@ public function onPointBuild(PointBuilderEvent $event): void
             'group'       => 'mautic.page.point.action',
             'label'       => 'mautic.page.point.action.urlhit',
             'description' => 'mautic.page.point.action.urlhit_descr',
-            'callback'    => [PointActionHelper::class, 'validateUrlHit'],
+            'callback'    => [$this->pointActionHelper, 'validateUrlHit'],
```

Comment:
I think this will have to be replaced with the `eventName` as it was done years ago for Form actions: https://github.com/mautic/mautic/pull/8186/files#diff-58a1e22576fb1ec60bd00a76d0b2f1408b61374c36d9d23e64d41cc0f84b1e4dR117

Reply:
For me that's a totally different scope. This PR targets to remove deprecated MauticFactory